scripts/build/ios-appstore.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+# Builds the Firezone iOS client for submitting to the App Store
+
+set -euo pipefail
+
+source "./scripts/build/lib.sh"
+
+# Define needed variables
+app_profile_id=8da59aa3-e8da-4a8c-9902-2d540324d92c
+ne_profile_id=0fccb78a-97c0-41b9-8c54-9c995280ea8e
+temp_dir=$(mktemp -d)
+archive_path="$temp_dir/Firezone.xcarchive"
+package_path="$temp_dir/Firezone.ipa"
+export_options_plist_path="$temp_dir/ExportOptions.plist"
+git_sha=${GITHUB_SHA:-$(git rev-parse HEAD)}
+project_file=swift/apple/Firezone.xcodeproj
+code_sign_identity="Apple Distribution: Firezone, Inc. (47R2M6779T)"
+
+if [ "${CI:-}" = "true" ]; then
+    # Configure the environment for building, signing, and packaging in CI
+    setup_runner \
+        "$IOS_APP_PROVISIONING_PROFILE" \
+        "$app_profile_id.mobileprovision" \
+        "$IOS_NE_PROVISIONING_PROFILE" \
+        "$ne_profile_id.mobileprovision"
+fi
+
+# Build and sign app
+insert_build_timestamp "$project_file/project.pbxproj"
+
+echo "Building and signing app..."
+xcodebuild archive \
+    GIT_SHA="$git_sha" \
+    CODE_SIGN_STYLE=Manual \
+    CODE_SIGN_IDENTITY="$code_sign_identity" \
+    APP_PROFILE_ID="$app_profile_id" \
+    NE_PROFILE_ID="$ne_profile_id" \
+    -project "$project_file" \
+    -skipMacroValidation \
+    -archivePath "$archive_path" \
+    -configuration Release \
+    -scheme Firezone \
+    -sdk iphoneos \
+    -destination 'generic/platform=iOS'
+
+# iOS requires a separate export step; write out the export options plist
+# here so we can inject the provisioning profile IDs
+cat <<EOF >"$export_options_plist_path"
```

Comment:
I decided to inline this file to consolidate build context in this script. That file was not used for debug builds.